tests/test_dataset.py

```diff
@@ -148,6 +148,19 @@ def test_get_fusion_dataset_to_latest_commit(self, accesskey, url):
 
         gas_client.delete_dataset(dataset_name)
 
+    def test_update_dataset(self, accesskey, url):
+        gas_client = GAS(access_key=accesskey, url=url)
+        dataset_name = get_dataset_name()
+        dataset_client = gas_client.create_dataset(dataset_name)
+        dataset_client.create_draft("v_test")
```

Comment:
You don't need to 'create_draft' and 'commit' since it is irrelevant with version control.